index.ts

```diff
@@ -90,6 +98,10 @@ if (conf.get('websocket.ws.enable')) {
     serverWS.listen(conf.get('websocket.ws.port'));
     serverWS.on('listening', (): void => {
         logger.info('ws server listening on', conf.get('websocket.ws.port') );
+        process.send({  // signal parent process that WS server is ready(for testing)
+            sType: 'WS',
+            message: 'server startup'
+        });
```

Comment:
As discussed, these separate `process.send` will be funnel-down into a single `Promise.all` call to be resolved and send back a signal to the parent process.